.github/workflows/cd-pipeline.yaml

```diff
@@ -112,50 +111,91 @@ jobs:
           VIB_ENV_S3_URL: s3://${{ secrets.AWS_S3_BUCKET }}/bitnami
           VIB_ENV_S3_USERNAME: ${{ secrets.AWS_ACCESS_KEY_ID }}
           VIB_ENV_S3_PASSWORD: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  update-index:
+    runs-on: ubuntu-latest
+    needs:
+      - vib-publish
+    name: Update charts index
+    outputs:
+      result: ${{ steps.update-index.outputs.result }}
+    if: ${{ needs.vib-publish.result == 'success' }}
```

Comment:
I think only having this conditional is the same as having the `needs: vib-publish`. With the `need`, this job will only run when `vib-publish` succeeds, so having an `if` to check the same thing is redundant.

Reply:
Removed, thanks!